ladybug/src/main/java/org/frankframework/ibistesttool/jdbc/LadyBugDataSourceFactory.java

```diff
@@ -117,6 +120,13 @@ private ObjectPool<PoolableConnection> createConnectionPool(PoolableConnectionFa
 	}
 
 	public DataSource getDataSource(String dataSourceName) {
+		// File storage will be used instead of database storage when dataSourceName is empty (see
+		// springIbisTestTool.xml) but Spring will still wire bean dataSource (calling this method) to bean scheduler
+		// (which will ignore it, see SchedulerFactoryBean.setDataSource()). Prevent parent class from throwing an
```

Comment:
In het framework wrappen we dit ding en halen we expliciet de datasource en tx manager setters er uit.